src/sentry/integrations/discord/actions/issue_alert/notification.py

```diff
@@ -14,7 +14,7 @@
 class DiscordNotifyServiceAction(IntegrationEventAction):
     id = "sentry.integrations.discord.notify_action.DiscordNotifyServiceAction"
     form_cls = DiscordNotifyServiceForm
-    label = "Send a notification to the {server} Discord server in the channel with ID: {channel_id} and show tags {tags} in the notification."
+    label = "Send a notification to the {server} Discord server in the channel with URL: {channel_url} and show tags {tags} in the notification."
```

Comment:
@17hogeju I think we'd support channel id OR channel URL, how hard would it be to do that?

Reply:
Easy 😄